integration-tests/messaging/src/main/java/org/apache/camel/quarkus/component/messaging/it/JmsRoutes.java

```diff
@@ -27,11 +27,10 @@
 
     @Override
     public void configure() throws Exception {
-        // jmsMessageType text / binary routes
-        from("jms:queue:typeTest?jmsMessageType=Text")
-                .toD("jms:queue:typeTestResult?jmsMessageType=${header.type}");
+        from("jms:queue:typeTest?concurrentConsumers=5")
+                .toD("jms:queue:typeTestResult");
```

Comment:
Maybe, we can get rid of toD then.

Reply:
Ah, thanks. Thought I caught those. I'll fix.

Reply:
Ok, fixing in another commit is good.